variants/backend-base/config/secrets.yml

```
@@ -0,0 +1,25 @@
+# Do NOT put secrets directly into this file. All secrets should be loaded from ENV!
+# Be sure to restart your server when you modify this file.
+
+default: &default
+  # Your secret key is used for verifying the integrity of signed cookies.
+  # If you change this key, all old signed cookies will become invalid!
+  # Make sure the secret is at least 30 characters and all random,
+  # no regular words or you'll be exposed to dictionary attacks.
+  # You can use `rails secret` to generate a secure secret key.
+  secret_key_base: "<%= ENV['RAILS_SECRET_KEY_BASE'] %>"
+  active_record_encryption_primary_key: "<%= ENV['ACTIVE_RECORD_ENCRYPTION_PRIMARY_KEY'] %>"
+  active_record_encryption_deterministic_key: "<%= ENV['ACTIVE_RECORD_ENCRYPTION_DETERMINISTIC_KEY'] %>"
+  active_record_encryption_key_derivation_salt: "<%= ENV['ACTIVE_RECORD_ENCRYPTION_KEY_DERIVATION_SALT'] %>"
```

We should be `ENV.fetch`ing these right?

```suggestion
  secret_key_base: "<%= ENV.fetch('RAILS_SECRET_KEY_BASE') %>"
  active_record_encryption_primary_key: "<%= ENV.fetch('ACTIVE_RECORD_ENCRYPTION_PRIMARY_KEY') %>"
  active_record_encryption_deterministic_key: "<%= ENV.fetch('ACTIVE_RECORD_ENCRYPTION_DETERMINISTIC_KEY') %>"
  active_record_encryption_key_derivation_salt: "<%= ENV.fetch('ACTIVE_RECORD_ENCRYPTION_KEY_DERIVATION_SALT') %>"
```

---

Yes for `secret_key_base` but I don't know if we want to **force** people to set the AR encryption stuff if they aren't actively using it 🤔

---

Had a think and I think we probably do want `fetch` here for everything because we are already basically forcing them to do the setup for encrypted credentials anyway. I have already updated this locally with some other changes so I won't accept this suggestion.